### ISSUE TITLE
Enum semicolon error

### ISSUE THREAD

**Issue by dsvoronin** [shyiko/ktlint#12]
ktlint formatter bug for semicolon in enums: 
```
    ONE("network_type_changed"),
    TWO("technical_unexpected_api_error"),
    THREE("technical_unexpected_api_response")  ; << SEMICOLON REMOVED HERE, AND IT BREAKS CODE

    override fun toString(): String = when (id) {
        null -> name.toLowerCase()
        else -> id
    }
```

**Comment by shyiko** [comment]
Fixed in 0.2.0.

**Comment by joel-coinbase** [comment]
Using version 0.30.0 I get the same error in this situation.

```
enum class SomeClass(val type: String) {
    TYPE_1("string_1"),
    ...
    TYPE_N("string_n"); << SEMICOLON NEEDS TO BE HERE, BUT LINT COMPLAINS

    companion object { ...
    }
}
```
